web/api/controller/data_controller.py

```
@@ -0,0 +1,28 @@
+    """
+    # background_tasks.add_task(news_service.sync_news, req.ticker)
+
+    return DataSyncResponse(
+        status="success",
+        message=f"{req.ticker} 뉴스 수집 작업이 백그라운드에서 시작되었습니다.",
+        collected_count=0
+    )
+
+
+@router.post("/sync/dart", response_model=DataSyncResponse, summary="DART 공시 수집 실행")
+async def sync_dart(req: DataSyncRequest):
```

Ruff (0.14.5)

Unused function argument: `req`

(ARG001)